Applications/JupyterHub.qmd

```diff
@@ -108,6 +116,36 @@ Working with a Notebook is pretty convenient and the supporting text should enab
 | M                 | Change cell to Markdown (text) mode |
 
 
+## Storage
+
+When you start the notebook, in the file navigation pane (the sidebar - #2) you will see three folders:
+
+* `/mystorage`
+* `/samples`
+* `/work`
+
+### `/mystorage`
+
+This is a persistent storage with 10GB of space, which is automatically created for each user during the first login to Jupyter Hub.
+The storage is hosted in the CloudFerro cloud and can be used to save notebooks between the sessions, store the results or uploaded data files.
+This storage area is preserved when you logout and the Jupyter kernel is shut down due to inactivity.
+It will be kept for up to 15 days from your last login, and you will receive a notification to log in to Jupyter Hub to reset the timer and keep the data preserved.
+If you do not log in, then after 15 days your files will be deleted from CloudFerro cloud storage.
+
+### `/samples` and `/work`
+
+These folders are recreated with every start of the Jupyter kernel.
+The samples folder is always up-to-date with the latest version of notebooks.
+If you make changes to the samples please make sure to save the updated notebook in the `/mystorage` area to make sure you can use it when you get back after period of incativity.
+
```

Comment:
what's the `work` folder for?

Reply:
and what happens with files/folders created outside of these three folders?

Reply:
The work folder will be removed as it has no specific use